app/components/Feed/index.js

```diff
@@ -16,13 +17,17 @@ export const activityRenderers = {
   event_register: require('./renders/event_register')
 };
 
-const Feed = ({ items }: { items: Array<AggregatedActivity> }) => (
+type Props = { items: Array<AggregatedActivity> };
+
+const Feed = ({ items }: Props) => (
   <div style={{ width: '100%' }}>
     {items.length ? (
       items.map((item, i) => {
         const renders = activityRenderers[item.verb];
         return renders ? (
-          <Activity aggregatedActivity={item} key={i} renders={renders} />
+          <ErrorBoundary>
+            <Activity aggregatedActivity={item} key={i} renders={renders} />
```

Comment:
shouldn't the `key` be passed to `ErrorBoundary` now?

Reply:
Yeah, testing it now